src/App.vue

```diff
@@ -2,32 +2,24 @@
   <div id="app">
     <button @click="getTotalSupply">Get Total MltiToken Supply</button>
     <form v-on:submit.prevent>
-      <input type="text" placeholder="Number of Tokens to Mint" v-model="tokenAmount" />
-      <button prevent="submit" @click="mintTokens">Mint Tokens</button>
+      <p>
+        <input type="text" placeholder="Number of Tokens to Mint" v-model="tokenAmount" />
+      </p>
+      <p>
+        <button prevent="submit" @click="mintTokens">Mint Tokens</button>
+      </p>
     </form>
     <hr />
     <pre v-if="response">{{ response | prettyJSON }}</pre>
   </div>
 </template>
 
 <script>
-import axios from 'axios';
-import axiosCookieJarSupport from '@3846masa/axios-cookiejar-support';
-import tough from 'tough-cookie';
+// [NEED TO FILL] The local address where the demo app will be served
+const BASE_URL = 'https://rjlrpdjikze7fn2vscxfuk365y.multibaas.com';
 
-axiosCookieJarSupport(axios);
-
-// Automatically store the cookie returned by the MultiBaas server
-axios.defaults.jar = new tough.CookieJar();
-// Automatically include the stored cookie in requests being made to the MultiBaas server
-axios.defaults.withCredentials = true;
-
-// The local address where the demo app will be served
-const baseURL = 'https://localhost:8080/';
-
-// Configure these values with your MultiBaas login information
-const apiUser = '';
-const apiPassword = '';
+// [NEED TO FILL] API key
+const API_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE1OTU4NTIzOTUsInN1YiI6IjEifQ.b6vrRuDLEQ-tfrRYq2v_NG43LiJXo1KMXzgYcZoCN7s';
```

Comment:
Eventually (soon?) we want to open source this. Let's keep API keys and URLs out of the code!

Reply:
Yep, I have already fixed https://github.com/curvegrid/vue-dapp-sample/pull/12/commits/5a1f9ab478247340a7d0cfb287eb729caf7a05cf I just forgot to push 😅

Reply:
Friendly reminder it's already public 😄

Reply:
Indeed!

Reply:
Yeah that is fine because I have already removed that instance and api key. 😅